docs/axelar/bridge-tokens-axelar.md

```diff
@@ -0,0 +1,172 @@
+    
+    - Using Foundry's `cast` command:
+
+    ```sh
+    WETH=0x7b79995e5f793a07bc00c21412e50ecae098e7f9
+    AXELAR_GATEWAY=0xAABdd46ba1B3147d0Cf6aCc9605a74fE8668fC74
+    cast send $WETH "approve(address guy, uint256 amount)" $AXELAR_GATEWAY $(cast to-wei 0.1) --private-key $PRIVATE_KEY --rpc-url $SEPOLIA_RPC_URL
+    ```
+
+    - Using [Etherescan](https://sepolia.etherscan.io/token/0x7b79995e5f793a07bc00c21412e50ecae098e7f9#writeContract), select **Connect to Web3** to connect your wallet. Expand the **approve** dropdown, enter `AxelarGateway`'s address (`0x832B2dfe8a225550C144626111660f61B8690efD`), and the amount of ETH to bridge under **deposit**. Click **Write** to perform the transcation.
+
+4. On Ethereum Sepolia, call `AxelarGateway.sendToken()` to bridge your WETH.
+
+    ```sh
+    AXELAR_GATEWAY=0xAABdd46ba1B3147d0Cf6aCc9605a74fE8668fC74
+    XRPL_DESTINATION= # your XRPL recipient address
+    cast send $AXELAR_GATEWAY "sendToken(string destinationChain, string destinationAddress, string symbol, uint256 amount)" "xrpl" $XRPL_DESTINATION "WETH" $(cast to-wei 0.1) --private-key $PRIVATE_KEY --rpc-url $SEPOLIA_RPC_URL
+    ```
+
+5. The destination address should receive the bridged ETH in a few minutes. You can check for a successful transaction on the [XRPL Testnet explorer](https://testnet.xrpl.org/).
+
+
+### Bridge ETH from XRPL to Ethereum
+
+1. Deposit ETH into the XRPL multisig account, using a `Payment` transaction.
+
+```javascript
+{
+    TransactionType: "Payment",
+    Account: user.address, // The user initiating the transfer
+    Amount: {
+        currency: "ETH",
+        value: "0.001", // = 10^15 wei - the amount of ETH you want to bridge, in ETH
+        issuer: "rfEf91bLxrTVC76vw1W3Ur8Jk4Lwujskmb",
+    },
+    Destination: "rfEf91bLxrTVC76vw1W3Ur8Jk4Lwujskmb", // Axelar's XRPL multisig account
+    Memos: [
+        {
+            Memo: {
+                MemoData: "605459C28E6bE7B31B8b622FD29C82B3059dB1C6", // your ETH recipient address, without the 0x prefix
+                MemoType: "64657374696E6174696F6E5F61646472657373", // hex("destination_address")
+            },
+        },
+        {
+            Memo: {
+                MemoData: "657468657265756D", // hex("ethereum")
+                MemoType: "64657374696E6174696F6E5F636861696E", // hex("destination_chain")
+            },
+        },
+        {
+            Memo: {
+                MemoData: "0000000000000000000000000000000000000000000000000000000000000000", // bytes32(0) indicates pure token transfer, without GMP
+                MemoType: "7061796C6F61645F68617368", // hex("payload_hash")
+            },
+        },
+    ],
+    ...
+}
+```
+
+2. Within a few minutes, the `destination_address` should receive the bridged ETH on Ethereum `Sepolia`, as WETH. You can check for the transaction on the [Ethereum Sepolia explorer](https://sepolia.etherscan.io/).
+
+
+### Bridge XRP from XRPL to Ethereum
+
```

Comment:
How about this diagram to explain the flow at a high level?
![XRPL Axelar XRP axlXRP](https://github.com/ripple/opensource.ripple.com/assets/6089916/551943da-7dbe-4b83-8b70-f86d4ad30a57)